resources/views/layouts/librenmsv1.blade.php

```diff
@@ -18,37 +18,40 @@
     @endif
 
     <link rel="manifest" href="{{ asset('images/manifest.json') }}" crossorigin="use-credentials">
-    <meta name="csrf-token" content="{{ csrf_token() }}">
-    <meta name="msapplication-config" content="{{ asset('images/browserconfig.xml') }}">
-    <meta name="theme-color" content="#ffffff">
+    <meta name="csrf-token" content="{{ csrf_token() }}" />
+    <meta name="msapplication-config" content="{{ asset('images/browserconfig.xml') }}" />
+    <meta name="theme-color" content="#ffffff" />
```

Comment:
adding some slashes, and deleting some :(

Reply:
depends on the tag, unclear which it should be.  I can remove them again.